src/components/ModelTableList/OwnerGroup.test.js

```diff
@@ -40,12 +40,13 @@ describe("OwnerGroup", () => {
       </MemoryRouter>
     );
     const tables = wrapper.find("MainTable");
-    expect(tables.length).toBe(5);
-    expect(tables.get(0).props.rows.length).toEqual(5);
-    expect(tables.get(1).props.rows.length).toEqual(6);
-    expect(tables.get(2).props.rows.length).toEqual(1);
-    expect(tables.get(3).props.rows.length).toEqual(3);
-    expect(tables.get(4).props.rows.length).toEqual(2);
```

Comment:
This is not a good way of testing. We should find a way to reference these without index.